pkg/render/fluentd_test.go

```diff
@@ -230,7 +230,7 @@ var _ = Describe("Tigera Secure Fluentd rendering tests", func() {
 		}
 
 		cfg.OSType = rmeta.OSTypeWindows
-		cfg.MetricsServerTLS = nil
+		cfg.FluentdKeyPair = nil
```

Comment:
I have noticed I have the following variables set locally:

```
		{Name: "LINSEED_ENABLED", Value: "true"},
		{Name: "LINSEED_ENDPOINT", Value: "https://tigera-linseed.tigera-elasticsearch.svc"},
		{Name: "LINSEED_CA_PATH", Value: c.cfg.TrustedBundle.MountPath()},

```

Reply:
What about them?